ramls/item.json

```diff
@@ -92,6 +92,10 @@
     "itemDamagedStatusDate": {
       "type": "string"
     },
+    "itemDamagedStatusDescription": {
```

Comment:
Hi @marcjohnson-kint, JIRA says that the new property should be named as `itemDamagedDescription`, but existing properties have  `itemDamagedStatusDate` and `itemDamagedStatusId` names, should the new property follow the name pattern and be named as  `itemDamagedStatusDescription`?

Reply:
I'll leave @marcjohnson-kint to be the final word here, but to me, `itemDamagedDescription` seems right (I'd prefer just `damageDescription`, since we already know it is an `item`, but again I'll defer to Marc). The description is of the damage to the item. The date is for the status change from whatever to `Damaged` or whatever the damaged status needs to be. The id also refers to the specific damage status, like `Damaged` or `Damaged by patron` or `Damaged in transit` or pretty much anything the library wants.

However, the description here only applies to the damage of the item, like `Page 5 missing` or `Coffee spilled on cover` or `Currently on fire`. This doesn't describe the status, but the item itself. I suppose the damage status could change several times due to a item with bad luck, but the description would continue to evolve/grow as more damage occurs or is repaired.

I'm actually surprised damage is not repeatable. Wait a second, according to scenario 2 in UIIN-593 (a linked story in the invstor JIRA), the elements are  "Not repeatable/Not required" for viewing, but for editing, scenario 1, the elements are "repeatable, not required". Maybe it is just me, but that is confusing. In UXPROD-1659, Charlotte asks Emma if a decision on whether or not damage is repeatable, but there is no response. UIIN-583 was created before this comment was added to UXPROD-1659. After some digging, it looks like UIIN-562 confirms that this has been decided "damage" to be a non-repeatable field.

I think we need to get the POs to do a better job coordinating and documenting clear requirements. UIIN-562 is about adding damage status types, not the overall feature. OK, enough ranting. My comment still stands about `itemDamagedDescription`. 😄

Reply:
> I'll leave @marcjohnson-kint to be the final word here, but to me, `itemDamagedDescription` seems right 

@bohdan-suprun I think I pretty much agree with @mreno-EBSCO 

> (I'd prefer just `damageDescription`, since we already know it is an `item`, but again I'll defer to Marc). 

I wasn't involved in the earlier work in this area. I'd hope that if I had been I might have designed it something like:

```
“damage” (or damaged, but that doesn't read as well to me) : {
 “statusId”: ...
 “description”: ...
 “lastChangedDate”: ....
}
```

However that comes with the retrospective coherence based upon the scenarios in this issue, so I might have fallen into similar pitfalls.

> The description is of the damage to the item. The date is for the status change from whatever to `Damaged` or whatever the damaged status needs to be. The id also refers to the specific damage status, like `Damaged` or `Damaged by patron` or `Damaged in transit` or pretty much anything the library wants.

> However, the description here only applies to the damage of the item, like `Page 5 missing` or `Coffee spilled on cover` or `Currently on fire`. This doesn't describe the status, but the item itself. I suppose the damage status could change several times due to a item with bad luck, but the description would continue to evolve/grow as more damage occurs or is repaired.

My interpretations of the requirements is that this is the description of the damage, not a description of the status of the damage.

To be consistent with the existing properties, that would mean naming it `itemDamagedDescription` which doesn't read well to me, but consistency and implicit grouping is likely more important than readability at this point.

Given you are unclear, please feel free to confirm with the PO by asking a question on the issue. If you don't feel comfortable doing that, let me know, and I can ask on your behalf.

> I'm actually surprised damage is not repeatable. Wait a second, according to scenario 2 in UIIN-593 (a linked story in the invstor JIRA), the elements are "Not repeatable/Not required" for viewing, but for editing, scenario 1, the elements are "repeatable, not required". Maybe it is just me, but that is confusing. In UXPROD-1659, Charlotte asks Emma if a decision on whether or not damage is repeatable, but there is no response. UIIN-583 was created before this comment was added to UXPROD-1659. After some digging, it looks like UIIN-562 confirms that this has been decided "damage" to be a non-repeatable field.

I'm both surprised and confused.

> I think we need to get the POs to do a better job coordinating and documenting clear requirements. UIIN-562 is about adding damage status types, not the overall feature. OK, enough ranting. My comment still stands about `itemDamagedDescription`. 😄

I think it's clear from the previous work that there has been some confusion in the requirements for this area.

For example, scenario 3 describes updating the date when the description changes. Unfortunately this property has been named `itemDamagedStatusDate` in the API, which is somewhat misleading now it's broader purpose is understood.

This is probably compatibility breaking technical debt, should we raise an issue for it?